core/src/main/java/org/apache/spark/util/collection/unsafe/sort/UnsafeInMemorySorter.java

```diff
@@ -19,8 +19,7 @@
 
 import java.util.Comparator;
 import java.util.LinkedList;
-
-import org.apache.avro.reflect.Nullable;
+import javax.annotation.Nullable;
```

Comment:
Why do we need this change?

Reply:
I suppose yes, this is an improvement because `Nullable` in that class is used to designate fields as nullable and not in any aspect related to Avro.